### PR TITLE
docs: Added validator `runtypes` to docs

### PR DESCRIPTION
**Pull request by AlexXanderGrib**
## 🎯 Changes

- Added [`runtypes`](https://www.npmjs.com/package/runtypes) to supported validators list

It works without any changes to code, cause it has interface similar to `scale-ts`

```typescript
export type ParserScaleEsque<TInput> = {
  assert(value: unknown): asserts value is TInput;
};

// runtypes/src/runtype.ts
export interface RuntypeBase<A = unknown> {
  /**
   * Verifies that a value conforms to this runtype. When given a value that does
   * not conform to the runtype, throws an exception.
   */
  assert(x: any): asserts x is A;
}
```

## ✅ Checklist

- [x] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [x] If necessary, I have added documentation related to the changes made.
- [x] I have added or updated the tests related to the changes made.

### CODE REVIEW

File: www/docs/server/validators.md [hunk 1]
```diff
@@ -363,6 +363,32 @@ export const appRouter = t.router({
...
+    .query(({ input }) => {
+      //      ^?
+      return {
+        greeting: `hello ${input.name}`,
+      };
+    }),
+});
+
+export type AppRouter = typeof appRouter;
+```
```
Comment (Nick-Lucas) [inline www/docs/server/validators.md:389]: Thank you for your contribution. We also have a paired tests file under ./packages/tests (apologies, away from PC so can't recall the exact file) which we like to add libraries to, this way we don't accidentally break compatibility in the future.

Could you please add it there too? I'll be happy to merge then 🙂

**Review by Nick-Lucas — CHANGES_REQUESTED** [review]

**Review by Nick-Lucas — APPROVED** [review]
LGTM, thank you for your contribution 🙂

**Review by Nick-Lucas — APPROVED** [review]

**Comment by Nick-Lucas** [comment]
@AlexXanderGrib you have some linting failures. `pnpm lint-fix` `pnpm format-write` solve most of these issues :)

**Comment by AlexXanderGrib** [comment]
@Nick-Lucas done. Lint passes

File: www/docs/server/validators.md [hunk 2]
```diff
@@ -363,6 +363,31 @@ export const appRouter = t.router({
...
+    .output(t.Record({ greeting: t.String }))
+    .query(({ input }) => {
+      //      ^?
+      return {
+        greeting: `hello ${input.name}`,
+      };
+    }),
+});
+
+export type AppRouter = typeof appRouter;
```
Comment (juliusmarminge) [inline www/docs/server/validators.md:388]: `t` & `t` :)

```suggestion
import { initTRPC } from '@trpc/server';
import * as T from 'runtypes';

const t = initTRPC.create();
const publicProcedure = t.procedure;

export const appRouter = t.router({
  hello: publicProcedure
    .input(T.Record({ name: T.String }))
    .output(T.Record({ greeting: T.String }))
    .query(({ input }) => {
      //      ^?
      return {
        greeting: `hello ${input.name}`,
      };
    }),
});

export type AppRouter = typeof appRouter;
```

You'll also need to add `runtypes` as a devDep to www for shiki to be able to compile.

I did it locally but couldn't push to your branch :/

**Review by Nick-Lucas — APPROVED** [review]

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**